sample-app/src/main/res/layout/activity_edit_text.xml

```diff
@@ -20,7 +20,6 @@
             android:cursorVisible="false"
             android:digits="1234567890"
             android:inputType="number"
-            android:maxLength="4"
```

Comment:
Can you please discard this particular change.
It may have helped you test the change, however, I don't want to change the examples as they currently are.
You can add your own activity or PinEntryEditText to the screen to demonstrate the new feature if you like.